File: 02-backend-booking-system/src/main/java/com/littlek4za/booking_system/dto/EventResponseDto.java
package com.littlek4za.booking_system.dto;

import java.time.Instant;

public record EventResponseDto(
    Long id,
    String userName,
    String eventName,
    String eventDescription,
    String eventLocationName,
    Boolean includePosition,
    Double latitude,
    Double longitude,
    String slotType,
    Instant createdAt
) {}

File: 03-frontend-booking-system/booking-system-web/src/app/common/event-response-dto.ts
export class EventResponseDto {
    id!:number;
    username!:string;
    eventName!:string;
    eventDescription?:string;
    eventLocationName!:string;
    includePosition!:boolean;
    latitude?:number;
    longitude?:number;
    slotType!: 'FIXED'|'FLEXIBLE'|'BUSINESS';
    createdAt!: string;
}
